src/main/java/teammates/storage/sqlapi/FeedbackResponsesDb.java

```diff
@@ -0,0 +1,87 @@
+package teammates.storage.sqlapi;
+
+import static teammates.common.util.Const.ERROR_CREATE_ENTITY_ALREADY_EXISTS;
+import static teammates.common.util.Const.ERROR_UPDATE_NON_EXISTENT;
+
+import java.util.UUID;
+
+import teammates.common.exception.EntityAlreadyExistsException;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.util.HibernateUtil;
+import teammates.storage.sqlentity.FeedbackResponse;
+
+/**
+ * Handles CRUD operations for feedbackResponses.
+ *
+ * @see FeedbackResponse
+ */
+public final class FeedbackResponsesDb extends EntitiesDb<FeedbackResponse> {
+
+    private static final FeedbackResponsesDb instance = new FeedbackResponsesDb();
+
+    private FeedbackResponsesDb() {
+        // prevent initialization
+    }
+
+    public static FeedbackResponsesDb inst() {
+        return instance;
+    }
+
+    /**
+     * Gets a feedbackResponse or null if it does not exist.
+     */
+    public FeedbackResponse getFeedbackResponse(UUID frId) {
+        assert frId != null;
+
+        return HibernateUtil.get(FeedbackResponse.class, frId);
+    }
+
+    /**
+     * Creates a feedbackResponse.
+     */
+    public FeedbackResponse createFeedbackResponse(FeedbackResponse feedbackResponse)
+            throws InvalidParametersException, EntityAlreadyExistsException {
+        assert feedbackResponse != null;
+
+        if (!feedbackResponse.isValid()) {
+            throw new InvalidParametersException(feedbackResponse.getInvalidityInfo());
+        }
+
+        if (getFeedbackResponse(feedbackResponse.getId()) != null) {
+            throw new EntityAlreadyExistsException(
+                    String.format(ERROR_CREATE_ENTITY_ALREADY_EXISTS, feedbackResponse.toString()));
+        }
+
+        persist(feedbackResponse);
+        return feedbackResponse;
+    }
+
+    /**
+     * Saves an updated {@code FeedbackResponse} to the db.
+     */
+    public FeedbackResponse updateFeedbackResponse(FeedbackResponse feedbackResponse)
+            throws InvalidParametersException, EntityDoesNotExistException {
+        assert feedbackResponse != null;
+
+        if (!feedbackResponse.isValid()) {
+            throw new InvalidParametersException(feedbackResponse.getInvalidityInfo());
+        }
+
+        if (getFeedbackResponse(feedbackResponse.getId()) == null) {
+            throw new EntityDoesNotExistException(ERROR_UPDATE_NON_EXISTENT);
+        }
+
+        return merge(feedbackResponse);
+    }
```

Comment:
Update not required in db layer anymore. Hibernate managed entities tracks changes and updates the db automatically.